Doc/library/pprint.rst

```diff
@@ -50,7 +50,13 @@ The :mod:`pprint` module defines one class:
    structure cannot be formatted within the constrained width, a best effort will
    be made.  If *compact* is false (the default) each item of a long sequence
    will be formatted on a separate line.  If *compact* is true, as many items
-   as will fit within the *width* will be formatted on each output line.
+   as will fit within the *width* will be formatted on each output line. If
+   *sort_dicts* is true (the default), dictionaries will be formatted with their
+   keys sorted alphabetically, otherwise they will be sorted by their insertion
+   order.
+
+   .. versionchanged:: 3.8
```

Comment:
Is not better write the last `.. versionchanged` after the previous `.. versionchanged`. I mean 3.4 before and then 3.8?

Reply:
This should be below as suggested and also I think this should be versionadded since this adds a new parameter to the signature.

Reply:
Thanks, I fixed it.

Reply:
> This should be below as suggested and also I think this should be versionadded since this adds a new parameter to the signature.

Grepping through the documentation it seems to me that `versionadded` is only used when a new object is added.

Reply:
Ah you might be right. Sorry for the noise. https://devguide.python.org/documenting/

> Similar to versionadded, but describes when and what changed in the named feature in some way (new parameters, changed side effects, platform support, etc.). This one must have the second argument (explanation of the change).